### PR TITLE
Documentation for config properties

### PR DESCRIPTION
**Pull request by hendrikebbers**
With the next version of OpenWebStart we will support to define properties at installation time. Next to this configurations in the UI can be deactivated. This documentation gives a short overview about the features.

### CODE REVIEW

File: _data/config-properties.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  lockable: true
+
+- name: 'ows.jvm.manager.server.allowFromJnlp'
+  description: 'Defines if a custom URL can be used to download a JVM. Such URL can be part 
+  of a JNLP file.'
+  lockable: true
+
+- name: 'ows.jvm.manager.vendor'
+  description: 'Defines a specifc JVM vendor. By doing so only JVMs from that vendor will 
+  be downloaded.'
```
Comment (sclassen) [inline _data/config-properties.yml]: Maybe give a hint to the special value `*`

File: _data/config-properties.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  lockable: true
+
+- name: 'ows.jvm.manager.updateStrategy'
+  description: 'When starting a JNLP application OpenWebStart can check if an updated JVM is 
+  available to run the application. This property defines how OpenWebstart behaves in the JVM check.'
```
Comment (sclassen) [inline _data/config-properties.yml]: List the valid values

File: predifined-and-locked-properties.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+headerText:
+permalink: /configuration/
+header:
+  image: webstart-2
+  text: Predefinied configuration for <span class="my-karakun">Open</span>WebStart
+hidden: true
+nav:
+  bottom: false
+---
+When installing OpenWebStart several properties can be prededined in a so called `response.varfile` file. The File must be passed to the installer by the following command:
```
Comment (sclassen) [inline predifined-and-locked-properties.md]: typo: `prededined` -> `predefined`

File: predifined-and-locked-properties.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+{% highlight properties %}
+# install4j response file for OpenWebStart 0.5.0-SNAPSHOT
+sys.adminRights$Boolean=false
+sys.fileAssociation.extensions$StringArray="jnlp","jnlpx"
+sys.fileAssociation.launchers$StringArray="313","313"
+sys.installationDir=/Applications/OpenWebStart
+sys.languageId=de
+{% endhighlight %}
+
+You can easily edit this file and add additional properties based on the table below. The initial content of the file should not be changed and new properties should always be added to the end of the file. After mutation a `response.varfile` file might look like this:
```
Comment (sclassen) [inline predifined-and-locked-properties.md]: change: `below` -> `above`

**Comment by hendrikebbers** [comment]
@sclassen will add the command for headless installation ;)

**Pull request merged**